### PR TITLE
Document how to build the project #34

### PR DESCRIPTION
**Pull request by skyrylyuk**
Integrate use 'gradle-intellij-plugin' for build plugin

### CODE REVIEW

**Comment by pbreault** [comment]
@skyrylyuk I can't get the plugin to build with this approach.


I tried jetbrain's plugin a while ago and it turned out to be insufficient for our purpose.

The reason is that adb idea depends on the android plugin that is distributed with android studio and intellij... which is not part of the dependencies that are fetched by the gradle plugin. If you take a look at what they are fetching, you will see that it only takes the core of intellij without any plugin.

So, basically, since we can't depend on the android plugin, we can't build. The correct approach would be for jetbrains to allow us to download an android studio zip and extract it.
